README.md

```diff
@@ -154,7 +159,12 @@ def source_handler(app, docname, source):
 ```
 
 #### Directive
+One of the harder parts of getting read the docs to work. Be ware of white space 
+and trailing new lines. A home-made directive like shown below *must* have 
+two new lines after it (which comments counting as well). 
```

Comment:
I don't understand what `which comments counting as well` means. Do you mean "with" instead of "which"?